Makefile

```diff
@@ -22,11 +22,11 @@ test:
 
 .PHONY: git-test
 git-test:
-	pytest evalml/ -n 8 --doctest-modules --cov=evalml --junitxml=test-reports/junit.xml --doctest-continue-on-failure
+	pytest evalml/ -n 4 --doctest-modules --cov=evalml --junitxml=test-reports/junit.xml --doctest-continue-on-failure
```

Comment:
Did we need to reduce the number of parallel processes because the github vms have less memory?

Reply:
It seems that having 8 processes causes one of them to crash during execution, and I do think it has something to do with there being less memory. It requires deeper digging though because I don't think it should require all the space it's been given